lambdamud-client/src/components/SignInForm.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import axios from 'axios';
+
+class SignInForm extends React.Component {
+  state = {
+    username: '',
+    email: '',
+    password: '',
+  };
+
+  // allows us to submit a new user and post it to the API
+  userSignIn = event => {
+    event.preventDefault();
+
+    console.log(this.state);
+
+    // Submit user information
+    const userInfo = {
+      'username': this.state.username,
+      'email': this.state.email,
+      'password': this.state.password,
+      
+    }
+
+    // POST axios call that submits a new user to login onto a server deployed on heroku
+    axios
+      .post('https://lambdamud-project-week.herokuapp.com/api/login/', userInfo)
+
+      .then(response => {
+        console.log(response);
+        // stores the token in local storage
+        localStorage.setItem('token', response.data.key);
+        // resets the state
+        this.setState({username: '', email: '', password: ''});
+      })
```

Comment:
I noticed that you have to reload your page or go to the main screen after signing in. This isn't ideal. You might consider using window.location.reload() call like I mentioned in the comment about your sign out functionality or you may consider routing your app to the mainscreen page after the axios call. I think it can be done by using this.props.history.push('url of your main page').